src/main/java/jenkins/plugins/office365connector/model/Action.java

```diff
@@ -0,0 +1,12 @@
+package jenkins.plugins.office365connector.model;
+
+import java.util.List;
+
+public interface Action {
```

Comment:
`Action` is very generic and very popular class name, how about making `ConnectorAction` or similar to avoid mismatch in class importing?

Reply:
Will take `CardAction` since the new way is not a `Connector` anymore, but a `PowerAutomate Workflow`.

Reply:
Done